src/__swaps__/screens/Swap/constants.ts

```diff
@@ -57,15 +58,27 @@ export const MAXIMUM_SIGNIFICANT_DECIMALS = 6;
 
 // /---- ⏱️ Animation configs ⏱️ ----/ //
 //
-export const buttonPressConfig = { duration: 160, easing: Easing.bezier(0.25, 0.46, 0.45, 0.94) };
-export const caretConfig = { duration: 300, easing: Easing.bezier(0.87, 0, 0.13, 1) };
-export const fadeConfig = { duration: 200, easing: Easing.bezier(0.22, 1, 0.36, 1) };
-export const pulsingConfig = { duration: 1000, easing: Easing.bezier(0.37, 0, 0.63, 1) };
-export const sliderConfig = { damping: 40, mass: 1.25, stiffness: 450 };
-export const slowFadeConfig = { duration: 300, easing: Easing.bezier(0.22, 1, 0.36, 1) };
-export const snappySpringConfig = { damping: 100, mass: 0.8, stiffness: 275 };
-export const snappierSpringConfig = { damping: 42, mass: 0.8, stiffness: 800 };
-export const springConfig = { damping: 100, mass: 1.2, stiffness: 750 };
+
+export const disableForTestingEnvironment = (config: {
+  duration?: number;
+  easing?: any;
+  damping?: number;
+  mass?: number;
+  stiffness?: number;
+}) => {
+  if (!IS_TESTING) return config;
```

Comment:
i think this will always return true if IS_TESTING is defined since it's a string. You probably want `IS_TEST` here